cfgov/v1/forms.py

```diff
@@ -102,22 +106,59 @@ class FilterableListForm(forms.Form):
     authors = forms.MultipleChoiceField(required=False, choices=[], widget=widgets.SelectMultiple(attrs=authors_select_attrs))
 
     def __init__(self, *args, **kwargs):
-        parent = kwargs.pop('parent')
-        hostname = kwargs.pop('hostname')
+        self.parent = kwargs.pop('parent')
+        self.hostname = kwargs.pop('hostname')
         super(FilterableListForm, self).__init__(*args, **kwargs)
-        page_ids = CFGOVPage.objects.live_shared(hostname).descendant_of(parent).values_list('id', flat=True)
-        self.set_topics(parent, page_ids, hostname)
-        self.set_authors(parent, page_ids, hostname)
+        page_ids = CFGOVPage.objects.live_shared(self.hostname).values_list('id', flat=True)
+
+        self.clean_categories()
+        self.set_topics(page_ids)
+        self.set_authors(page_ids)
+
 
+    def clean_categories(self):
+        """ This is a (hopefully) temporary solution for dealing w/ the fact
+        that we show Blog and Reports as options for filtering, but
+        aren't categories themselves. Rather, they consist of sub-categories,
+        but since we aren't showing those sub-categories to the end user,
+        selecting the parent category is equivalent to all of them
+        getting checked. The mapping of Blog and Reports to their
+        respective categories exists in cfgov/v1/util/ref.py
+        """
```

Comment:
@richaagarwal We only offer the opportunity to filter by parent category (Blog, News, etc) on the Activity Log, so it's not a "standard" option. The Activity Log page is older flapjack-era code and pre-dates the other filterable lists, so it was probably created with slightly different logic. As things stand, it seems that the "parent category" as you call it, comes from the site section in which the browse page lives and not some other assignment.

Worryingly, since the PR has been merged, the Activity log page on build is missing these parent categories. What needs to happen to fix that?